pkg/network/ebpf/c/protocols/events-types.h

```diff
@@ -0,0 +1,36 @@
+#ifndef __USM_EVENTS_TYPES_H
+#define __USM_EVENTS_TYPES_H
+
+#include <linux/types.h>
+
+#define BATCH_BUFFER_SIZE (4*1024)
+#define BATCH_PAGES_PER_CPU 3
```

Comment:
this has to be big enough to let new entries be buffered _while_ a complete batch gets flushed
in _theory_ this could be one because `netif_receive_skb` should execute roughly as often as the the socket filter programs, which means that we should almost always have `batch_enqueue` calls intertwined with `flush_batch` calls, but having a page size of 3 affords us to be less reliant on these assumptions